sdk/textanalytics/azure-ai-textanalytics/tests/testcase.py

```diff
@@ -5,6 +5,7 @@
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
+from re import L
```

Comment:
do we need this?

Reply:
ooh, looks like pylance snuck in an autoimport here, will remove